src/pdfkit.cs

```diff
@@ -434,6 +438,7 @@ interface PdfDocumentWriteOptions {
 
 		string OwnerPassword { get; set; }
 		string UserPassword { get; set; }
```

Comment:
Is missing the availability attires which should be the same as those of the key.

Reply:
without them the developer won't know they are not available (since the keys are not public)

and they (internal) keys also needs them otherwise introspection would not be able to do its job :)

Reply:
Updated :)